nilearn/datasets/func.py

```diff
@@ -2009,6 +2009,14 @@ def fetch_development_fmri(n_subjects=None, data_dir=None, resume=True,
         The number of subjects to load. If None, all the subjects are
         loaded. Total 155 subjects.
 
+    reduce_confounds: bool, optional (default True)
```

Comment:
Not so strong in my comment but just curious for keeping default to True ?

Reply:
I think this is there more reasonable option that does not require further interaction with the confounds